nbs/02_match_class.ipynb

```diff
@@ -21,7 +21,16 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        """Given a pair of node names, source and destination, return the name of the edge
</pre><p>kind of overkill to give this much of a docstring to such a small helper function, but since you already wrote it, its fine.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>